### PR TITLE
Config

### PR DESCRIPTION
**Pull request by jim-portegies**
* Move config file to a different place
* Add functionality to automatically find sertop
* Let the user choose a path to sertop

### CODE REVIEW

**Comment by davidot** [comment]
Tests are failing since the directories are Windows specific.
I fixed it but maybe we should use `mock-fs` in the future?
Probably easier than doing our own fs replacement thing

File: src/io/writeconfiguration.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            }
+          });
+    }).catch((err) => {
+      console.log('Error when reading configuration file prior to updating');
+      reject(err);
+    });
+  });
+}
+
+export {updateConfiguration};
```
[the remark this answers is not in the document]
Comment (jim-portegies) [reply src/io/writeconfiguration.js]: Good point, I merged the readconfiguration and writeconfiguration files. I kept separate test files for now.

**Comment by jim-portegies** [comment]
Concerning the linux wrapper: For some reason the linux wrapper was still the old version, I replaced it with the new one.
Concerning the mock filesystem: I agree that using mock-fs would be a good idea.

File: src/io/writeconfiguration.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidot) [reply src/io/writeconfiguration.js]: Ok, then everything seems good to me.

**Pull request merged**